app/src/main/kotlin/com/simplemobiletools/flashlight/views/SleepTimer.kt

```diff
@@ -0,0 +1,95 @@
+package com.simplemobiletools.flashlight.views
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.simplemobiletools.commons.compose.extensions.MyDevices
+import com.simplemobiletools.commons.compose.theme.AppThemeSurface
+import com.simplemobiletools.commons.compose.theme.SimpleTheme
+import com.simplemobiletools.flashlight.R
+
+@Composable
+internal fun SleepTimer(
+    modifier: Modifier = Modifier,
+    timerText: String,
+    onCloseClick: () -> Unit
+) {
+    Row(
+        modifier = modifier
+            .border(
+                width = 1.dp,
+                color = Color.Gray,
```

Comment:
We have predefined Colors for gray variants, please pick the one closest to it or see which one it was in the XML